docs/customization.rst

```diff
@@ -781,6 +793,92 @@ Lastly, make sure the project's documentation ``conf.py`` has the following line
 
     templates_path = ["_templates"]
 
+Adding a comment system
+-----------------------
+
+.. _Giscus: https://giscus.app/
+
+This theme supports using a third-party comment system of your choice at the bottom of any page.
+There are several services that can deliver an integrated comment system, but the following example
+demonstrates using Giscus_ which is Open Source and built on Github's Discussions feature.
+
+1. Ensure these requisites are completed:
+   
+   - Install the `Giscus Github App <https://github.com/apps/giscus>`_ and grant access to the
+     repository that should host comments as GitHub discussions. Note that this can be a repository
+     different from your documentation.
+   - Visit Giscus_ and generate the snippet through their configuration tool to load the comment
+     system. Copy the snippet for the next step. The resulting snippet should look similar to this:
+     
+     .. code-block:: html
+
+         <script
+           src="https://giscus.app/client.js"
+           data-repo="<username>/<repository>"
+           data-repo-id="..."
+           data-category="..."
+           data-category-id="..."
+           data-mapping="title"
+           data-reactions-enabled="1"
+           data-emit-metadata="1"
+           data-theme="light"
+           data-lang="en"
+           crossorigin="anonymous"
+           async>
+         </script>
+2. Override the theme's partials/comments.html template (blank by default) in your documentation
```

Comment:
It seems rather unfortunate to ask the user to add all of this boilerplate.  But I see that this example comes from mkdocs-material.

Reply:
Yeah. I'm not sure how much would change if using different comment systems (like Discus), but doing it this way allows the user more flexibility when following the docs for the chosen comment system.